lib/routes/actionsController.js

```diff
@@ -113,7 +113,6 @@ function PostAction(req, resp) {
     } else {
         delay = config.slaveDelay;
     }
-    // TODO: Tricky ... To refactor?
```

Comment:
Why this comment has been removed? Probably other developer put this here for some reason ;)

Reply:
This comment has been deleted due to one of the rules of the maintainability axis (ISO 25010).

TODO tags are commonly used to mark places where some more code is required, but which the developer wants to implement later.
Sometimes the developer will not have the time or will simply forget to get back to that tag.
This rule is meant to track those tags and to ensure that they do not go unnoticed.

More info: http://cwe.mitre.org/data/definitions/546.html

Reply:
In the case of not using a TODO marck, which other mechanism do you propose to track the need of a refactor mentioned by the developer here?

Reply:
I have reviewed it and it is an issue of INFO level, it is just a suggestion. Sorry for the inconvenience.

Reply:
Don't worry ;)

Thus, I understand the change will be rolled-back.

Reply:
Fixed in 3020aab9fb0f1a72ce0254cc6bc7db8401d13359